transport/conn.go

```diff
@@ -343,3 +344,19 @@ func (c *Conn) sendRequest(req frame.Request, compress, tracing bool) (frame.Res
 
 	return resp.Response, resp.Err
 }
+
+func (c *Conn) registerEvents(e []frame.EventType) (responseHandler, error) {
+	h := make(responseHandler)
+	c.r.h[eventStreamID] = h
+	req := &Register{EventTypes: e}
+
+	res, err := c.sendRequest(req, false, false)
+	if err != nil {
+		return nil, fmt.Errorf("registering for events: %w", err)
+	}
+	if _, ok := res.(*Ready); !ok {
```

Comment:
To keep consistency with
https://github.com/mmatczuk/scylla-go-driver/blob/main/transport/conn.go#L354
and
https://github.com/mmatczuk/scylla-go-driver/blob/main/transport/conn.go#L343
can we make this
`if _, ok := res.(*Ready); ok {`
with changed subsequent returns?